net/filters.go

```diff
@@ -9,6 +9,7 @@ import (
 func newPrivateIPFilters() (privateIPs *ma.Filters, err error) {
 	privateCIDRs := []string{
 		"10.0.0.0/8",
+		"127.0.0.1/0",
```

Comment:
Did that work?  If it did, the library must be treating `/0` as if the user wanted `/32`.  The entire `127.0.0.0/8` space is reserved, so that's what I would put down. Lots of linux hosts have multiple loopbacks for various reasons.

Reply:
right, yeah i think this only worked because the external address gets manually added after: https://github.com/AthanorLabs/atomic-swap/blob/master/net/host.go#L148 will update, thanks for catching this!